node/src/components/consensus/protocols/highway.rs

```diff
@@ -42,6 +43,14 @@ use crate::{
 /// even if eras are longer than this.
 const MAX_ENDORSEMENT_EVIDENCE_LIMIT: u64 = 10000;
 
+/// The timer for creating new units, as a validator actively participating in consensus.
+const TIMER_ID_ACTIVE_VALIDATOR: TimerId = TimerId(0);
+/// The timer for adding a vertex with a future timestamp.
+const TIMER_ID_VERTEX_WITH_FUTURE_TIMESTAMP: TimerId = TimerId(1);
+
+/// The action of adding a vertex from the `vertices_to_be_added` queue.
+const ACTION_ID_VERTEX: ActionId = ActionId(0);
```

Comment:
If we decide to have `TimerId` be an enum, maybe it makes sense for this to be an enum too?